packages/runtime/container-runtime/src/gc/gcConfigs.ts

```diff
@@ -98,41 +93,33 @@ export function generateGCConfigs(
 		}
 	}
 
-	// The persisted GC generation must indicate Sweep is allowed for this document,
+	// The persisted GC generation must indicate Sweep is enabled for this document,
 	// according to the GC Generation option provided this session.
-	// Note that if no generation option is provided, Sweep is allowed for any document.
-	const sweepAllowed = shouldAllowGcSweep(
+	// Note that if no generation option is provided, Sweep is enabled for any document.
+	const sweepEnabled = shouldAllowGcSweep(
```

Comment:
Can we be consistent with "allowed" meaning for the container and "enabled" meaning for the session?  i.e. revert these changes

Reply:
Changed it.